application/src/main/java/org/togetherjava/tjbot/commands/tags/TagCommand.java

```diff
@@ -62,4 +70,19 @@ public void onSlashCommand(SlashCommandInteractionEvent event) {
         }
         message.queue();
     }
+
+    @Override
+    public void onAutoComplete(CommandAutoCompleteInteractionEvent event) {
+        if (event.getName().equals(COMMAND_NAME)
```

Comment:
This is useless

Reply:
yes, i figured that out already, will remove it